src/commands/help.js

```diff
@@ -37,5 +37,6 @@ module.exports = {
       }
     });
     await message.author.send({ embed: helpEmbed });
+    await message.channel.send('Help message sent!');
```

Comment:
I don't think we need the 2nd await here

Reply:
You are correct - tested without and it still functioned as intended.